src/plugins/scriptedwizard/resources/plugins/wizard.script

```diff
@@ -289,30 +289,6 @@ function SetupProjectBuildOptions(project)
         project.AddIncludeDir(_T("$(#cb)\\include"));
         project.AddIncludeDir(_T("$(#cb)\\sdk\\wxscintilla\\include"));
         project.AddIncludeDir(_T("$(#cb)\\include\\tinyxml"));
-        if (WxVersion == 0)
-        {
-            project.AddIncludeDir(_T("$(#wx.include)"));
-            project.AddIncludeDir(_T("$(#wx.lib)\\gcc_dll\\mswu"));
-
-            project.AddLinkLib(_T("wxmsw28u"));
-            project.AddLibDir(_T("$(#wx.lib)\\gcc_dll"));
-        }
-        else if (WxVersion == 1)
-        {
-            project.AddIncludeDir(_T("$(#wx30.include)"));
-            project.AddIncludeDir(_T("$(#wx30.lib)\\gcc_dll\\mswu"));
-
-            project.AddLinkLib(_T("wxmsw30u"));
-            project.AddLibDir(_T("$(#wx30.lib)\\gcc_dll"));
-        }
-        else if (WxVersion == 2)
-        {
-            project.AddIncludeDir(_T("$(#wx31.include)"));
-            project.AddIncludeDir(_T("$(#wx31.lib)\\gcc_dll\\mswu"));
-
-            project.AddLinkLib(_T("wxmsw31u"));
-            project.AddLibDir(_T("$(#wx31.lib)\\gcc_dll"));
-        }
 
         project.AddLibDir(GetBaseDevelPath(WxVersion));
```

Comment:
i forget this...
I made a new commit that moves the library and search path to target level. Probably it would not be needed to move the library, but now they are in the same spot and there is no confusion.